src/qt/locale/bitcoin_zh_TW.ts

```diff
@@ -2577,8 +2577,8 @@ Note:  Since the fee is calculated on a per-byte basis, a fee of "100 satoshis p
         <translation><numerusform>预计在等待 %n 个区块后会有第一个确认。</numerusform></translation>
```

Comment:
This sentence is simplified Chinese.Translate to Traditional Chinese.
```suggestion
        <translation><numerusform>交易預計將在第 %n 個區塊後開始確認。</numerusform></translation>
```